app/scripts/metamask-controller.js

```diff
@@ -219,6 +221,21 @@ export default class MetamaskController extends EventEmitter {
       state: initState.CurrencyController,
     });
 
+    const tokenListMessenger = controllerMessenger.getRestricted({
+      name: 'TokenListController',
+    });
+    this.tokenListController = new TokenListController({
+      chainId: hexToDecimal(this.networkController.getCurrentChainId()),
+      useStaticTokenList: this.preferencesController.store.getState()
+        .useStaticTokenList,
+      onNetworkStateChange: this._onModifiedNetworkStateChange.bind(this),
```

Comment:
Could we just inline this method rather than creating a new method on the controller? I think it would be easier to understand its one-off usage if it was just all here. Also, unless the TokenListController needs the entire state, I'd suggest just passing the fields required back here.